student-store-express-api/app.js

```diff
@@ -1 +1,87 @@
-// YOUR CODE HERE
+const express = require('express');
+const morgan = require('morgan');
+const cors = require('cors')
+const fs = require('fs')
+
+const app = express();
+app.use(morgan("tiny"));
+app.use(express.json());
+app.use(cors())
+
+const JSONFilePath = './data/db.json';
+
+app.get('/', (req, res) => {
+    res.status(200).json({ ping: 'pong' });
+});
+
+app.get('/store', (req, res) => {
+    fs.readFile(JSONFilePath, 'utf8', (err, data) => {
+        const jsonData = JSON.parse(data);
+        res.status(200).json(jsonData);
+    })
+});
+
+app.post('/checkout', (req, res) => {
+    // Receive the request with the JSON body
+    // Extract the productCart attribute
+    // Iterate over every product in the cart
+        // For every product in the cart, look for price in database
+        // Add price times X quantity amount for given item
+    // Add additional 8.75% tax to this total
+    // Check last purchase and make new id for this purchases
+    // Build another JSON object with id, name, email, order, total, and createdAt
+    // Append this purchase to the db.json file
+    // Send back this JSON object and status 200 the request
+
```

Comment:
really good use of code comments; good job for handling the calculations in the backend :)